binding.gyp

```diff
@@ -138,6 +139,15 @@
           ],
           'conditions': [
             ['OS=="win"', {
+              'copies': [{
+                'destination': '<(PRODUCT_DIR)',
+                'files': [
+                  '<(jpeg_root)/bin/jpeg62.dll',
+                ]
+              }],
+              'include_dirs': [
+                '<(jpeg_root)/include'
+              ],
               'libraries': [
                 '-l<(GTK_Root)/lib/jpeg.lib'
```

Comment:
@zbjornson You're right, `jpeg.lib` isn't in GTK, so need the one from `libjpeg-turbo`.

That compiles. But... there's a nasty issue.

`img.src = fs.readFileSync('foo.jpg')` works
`img.src = foo.jpg` doesn't work

The problem is that libjpeg-turbo was compiled against `msvcrt.lib`, whilst Visual Studio uses `msvcrt{VS_VERSION}.lib`... Apparently this makes `jpeg_read_header` hang / crash... See http://stackoverflow.com/questions/4092251/alternatives-to-jpeg-read-header-libjpeg

I'll try fix `Image.cc` ... but in any case: it seems more involved then previously thought ;-)